test/simplemultisig.js

```diff
@@ -15,15 +15,22 @@ contract('SimpleMultiSig', function(accounts) {
   let acct
   let lw
 
-let createSigs = function(signers, multisigAddr, nonce, destinationAddr, value, data) {
-  
+let createSigs = function(signers, multisigAddr, nonce, destinationAddr, value, data, doSort) {
+
   let input = '0x19' + '00' + multisigAddr.slice(2) + destinationAddr.slice(2) + leftPad(value.toString('16'), '64', '0') + data.slice(2) + leftPad(nonce.toString('16'), '64', '0')
   let hash = solsha3(input)
 
   let sigV = []
   let sigR = []
   let sigS = []
 
+  //Put addresses in increasing order
+  if (doSort) {
+    signers.sort(function(a, b) {
+      return parseInt(a) - parseInt(b);
```

Comment:
In fact you can do that before even passing it into the function, and you may not need the extra `doSort` parameter.